core/two-steps.ts

```diff
@@ -0,0 +1,24 @@
+import { router } from '@slowreader/core'
+import { atom } from 'nanostores'
+
+export let secondStep = atom<boolean>(false)
+
+export function showSecondStep(): void {
+  secondStep.set(true)
+}
+
+export function showFirstStep(): void {
+  secondStep.set(false)
+}
+
+router.subscribe(route => {
```

Comment:
The logic looks similar to `backRoute`. What if we will replace `route.route === 'categories'` to `route.route === 'categories' && route.params.feed` in `backRoute` and here will use:

```js
backRoute.subscribe(url => {
  if (url) {
      showSecondStep()
  } else {
      showFirstStep()
  }
})
```